exampleSite/config.toml

```diff
@@ -140,7 +140,7 @@ ignoreErrors = ["error-remote-getjson"]
           avatarURL = "/images/avatar.png"
           # title shown in home page (HTML format is supported)
           title = ""
-          # subtitle shown in home page
+          # subtitle shown in home page (HTML format is supported)
```

Comment:
Can you add please `{{< version 0.1.1 changed >}}` to `config.toml` and `theme-documentation-basics/index.<LANG>.md` files?

Please take a look at [this PR](https://github.com/sunt-programator/CodeIT/pull/45/files#diff-1b323c3b150b637e63390b60e374400b9b58ed00ceaa944e1a0d4f335900100c).